app/wama.web/Views/AdminConsole/UserTool/ViewCertifications.cshtml

```diff
@@ -3,57 +3,29 @@
 @using WAMA.Web.Controllers;
 @using WAMA.Web.Model;
 @using WAMA.Core.Extensions;
-
-@model IEnumerable<CertificationViewModel>
```

Comment:
Once you pass in the data properly, there'll be no need for removing all the code that was removed.